website/docs/r/rest_managed.markdown

```diff
@@ -0,0 +1,87 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+layout: "aci"
+page_title: "ACI: aci_rest_managed"
+sidebar_current: "docs-aci-resource-rest-managed"
+subcategory: ""
+description: |-
+  Manages ACI Model Objects via REST API calls. This resource can only manage a single API object and its direct children. It is able to read the state and therefore reconcile configuration drift.
+---
+
+# aci_rest_managed (Resource)
+
```

Comment:
do we want to add some info to tell user in which case use this resource while in other cases use aci_rest resource?

Reply:
We will recommend the use of aci_rest_managed whenever people ask but I don't think we should add anything to the documentation of the resource itself.